### ISSUE TITLE
NGN Repo?

### ISSUE THREAD

**Issue by coreybutler** [coreybutler/questions#1]
Hi @ngn - I noticed you moved everything to gitlab and don't appear to be using Github anymore. Is there any chance you'd be willing to part with the @ngn handle? I have an OSS project called NGN that would really benefit from having a clear github org of that name.

Sorry to ping you this way... wasn't sure how else to reach you. I'm happy to take this offline if you'd prefer to not have any kind of presence/responses on Github. I can be reached at corey@coreybutler.com.

**Comment by ngn** [comment]
Hi @coreybutler, I'm sorry but I can't really part with my account, despite what happened to GitHub. I've built an identity around it for the past 8 years and accumulated quite a few "watch" and "follow" connections and I still use GitHub to keep an eye on the feed. Besides, scattered on the Internet there are some links to my projects, particularly to ngn/apl. I don't want to leave them dangling, so I must retain control here for as long as GitHub still exists.

**Comment by coreybutler** [comment]
Fair enough, and thanks for replying. In case it makes a difference (and for the sake of clarity), I'm not asking for the account, I'm merely looking to obtain the handle in order to create a new Github organization under that name. I'm asking if you'd be willing to rename your account in order to free up the ngn name, not give up your account. Nothing is lost when an account is renamed. All watches/follows/repos/stars/etc are maintained, and it does not affect the Github feed. Even history like comments/issues you've participated in/mentioned in are automatically handled.

I do understand where you're coming from... I have hundreds of followers, tens of thousands of stars (collectively), and millions of users using my repos. The projects have reached a point where they really need the attention of an entire community of developers, not just me - thus why I'm looking to setup an NGN organization.
